### DISCUSSION TITLE
Qt 5.15.12 Licence Requirements

### DISCUSSION THREAD

**Discussion by rubikstriangle** [AcademySoftwareFoundation/OpenRV#480] [Build Issues] [answered]
Is a paid license now required for installing Qt 5.15.12?  I installed qt for building openRV about a year ago and I don't remember needing a paid license.  I was able to get it going on a 10-day trail but once the trial is up how can I install QT?  Perhaps I'm doing something wrong? Are there any alternative suggestions for installing qt for an OpenRV build besides qt.io?

I just tried building qt from source but it is taking a really long time, not sure I want to go that route unless I really have to.

**Comment by geffrak** [comment, marked as answer]
Qt provides pre-compiled 5.15.2 binaries for everybody, and pre-compiled 5.15.16 for owners of commercial licences.

They provide the sources of 5.15.x 1 year after the release.

If you don't want to pay for Qt or compile it, I suggest you to pull 5.15.2 from the archives section of the Qt online installer.
